google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -27,7 +27,8 @@
 import com.google.common.collect.ImmutableSet;
 import java.util.Set;
 
-public class BigQueryOptions extends ServiceOptions<BigQuery, BigQueryOptions> {
+public class BigQueryOptions extends ServiceOptions<BigQuery, BigQueryOptions>
```

Comment:
Isn't it the Client class that needs to be autocloseable?

Reply:
+1 - this doesn't help users much.